.devcontainer/devcontainer.json

```diff
@@ -56,44 +41,22 @@
         "yzane.markdown-pdf",
         "yzhang.dictionary-completion",
         "yzhang.markdown-all-in-one",
-        "joshx.workspace-terminals",
-        "takumii.markdowntable",
-        "chdsbd.github-code-owners"
-      ]
+        "zoma.vscode-auto-open-workspace"
+      ],
+      "settings": {
+        "[makefile]": {
+          "editor.detectIndentation": false,
+          "editor.insertSpaces": false
+        },
+        "autoOpenWorkspace.enableAutoOpenIfSingleWorkspace": true,
+        "editor.formatOnSave": true,
+        "extensions.ignoreRecommendations": true,
+        "files.insertFinalNewline": true
+      }
     }
   },
-  "features": {
-    "ghcr.io/devcontainers/features/aws-cli:1": {
-      "version": "latest"
-    },
-    "ghcr.io/devcontainers/features/docker-in-docker:2": {
-      "moby": true,
-      "azureDnsAutoDetection": true,
-      "installDockerBuildx": true,
-      "installDockerComposeSwitch": true,
-      "version": "latest",
-      "dockerDashComposeVersion": "latest"
-    },
-    "ghcr.io/devcontainers/features/node:1": {
-      "nodeGypDependencies": true,
-      "version": "lts",
-      "nvmVersion": "latest"
-    },
-    "ghcr.io/devcontainers/features/github-cli:1": {
-      "installDirectlyFromGitHubRelease": true,
-      "version": "latest"
-    },
-    "ghcr.io/devcontainers/features/common-utils:2": {
-      "installZsh": true,
-      "configureZshAsDefaultShell": true,
-      "installOhMyZsh": true,
-      "installOhMyZshConfig": true,
-      "upgradePackages": true,
-      "username": "automatic",
-      "userUid": "automatic",
-      "userGid": "automatic"
-    }
-  }
-  // Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
-  // "remoteUser": "root"
+  "forwardPorts": [4000],
+  "image": "ghcr.io/nhsdigital/nhs-notify-template-repository:latest",
```

Comment:
This image needs updating with the installs in the Dockerfile. The accessibility tests can't run locally. 

Unsure if they still run in the pipeline.

Reply:
Seems fine when running in CI. Perhaps we just have this run in CI and ignore trying to run it locally?

Reply:
Can you do a PR please to https://github.com/NHSDigital/nhs-notify-repository-template/blob/main/src/jekyll-devcontainer/src/.devcontainer/Dockerfile with required changes.

Going forward will look at creating specific images per subdomain team, but for now will keep in one as we build up a base line